src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
@favonia I don't want to solve problems with the MLton tooling by changing my code...

We should probably turn off the unused variable warning, because they appear to be negatively useful.